config/file.go

```diff
@@ -92,6 +92,16 @@ func resolveConfigFilePath(path string) (string, error) {
 	return "", fmt.Errorf("failed to find config file %s", path)
 }
 
+// resolveFilePath uses the name if name is absolute path.
+// otherwise returns the combined path/name
+func resolveFilePath(path string, name string) string {
```

Comment:
2/5, but I'm wondering if this belongs as a method on `FileStore`, avoiding the need to pass `path` and just being able to access `fs.path` directly.

Reply:
Agreed